gyp/pylib/gyp/input.py

```diff
@@ -236,7 +236,11 @@ def LoadOneBuildFile(build_file_path, data, aux_data, includes, is_target, check
             # But since node-gyp produces ebcdic files, do not use that mode.
             build_file_contents = open(build_file_path, "r").read()
         else:
-            build_file_contents = open(build_file_path, "rU").read()
+            if sys.version_info > (3, 7) and sys.platform == "win32":
```

Comment:
Please resync this code with master.

Reply:
Because is misspelled.

Reply:
when i merged the error occures: `binding.gyp not found`. aslo there is no `hello_world` folder in node_modules

Reply:
>Please resync this code with master
@cclauss 
done)